fossas/fossa-cli
# Make verify sha_256 work for fossa cl versions before 3.4.7 on linux.

# Overview

An earlier PR broke `install-latest.sh` because it would try to download the wrong checksum file to check the archive with. 

## Acceptance criteria

* When installing versions of fossa before 3.4.7 on linux, we should successfully check the sha_256 rather than failing.

## Testing plan

1. Clone the master branch onto a linux system (or in a linux container with something like `docker run -it`)
2. Run `./install-latest.sh v3.2.3` and notice that it fails when trying to fetch the checksum file.
3. Check out this branch and run `./install-latest.sh v3.2.3` and see that it successfully installs the right version without an error.
4. Try `./install-latest.sh` and see that it installs the latest version correctly still.

## Risks

I think the main risk is if I've missed something in my version comparison function. 

## References

## Checklist


I didn't update the changelog since the broken version was never actually released. We had a customer using the script from our master branch.
